keps/sig-node/2403-pod-resources-allocatable-resources/README.md

```diff
@@ -207,67 +242,144 @@ To a vendor changes in the API should always be backwards compatible.
-Feature only collects data when requests comes in, data is then garbage collected. Data collected is proportional to the number of pods on the node.
+###### Will enabling / using this feature result in any new API calls?
+
+No
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No. The feature is not affecting hot code paths in the kubelet, and just give access to cached data already computed by the kubelet for internal bookkeeping.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+Negligible amount of CPU and memory, because the endpoint queries existing data structures inside the kubelet.
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No, because the endpoint queries existing data structures inside the kubelet.
 
 ### Troubleshooting
 
-* **How does this feature react if the API server and/or etcd is unavailable?**: No effect.
-* **What are other known failure modes?** feature gate disabled: the API will always return a well-known error. In normal operation, the API is expected to never return error and always return a valid response, because it utilizes internal kubelet data which is always available. Bugs may lead to the API to return unexpected errors, or to return inconsistent data. Consumers of the API should treat unexpected errors as bugs of this API.
-* **What steps should be taken if SLOs are not being met to determine the problem?** N/A
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+No impact, the feature is node-local
+
+###### What are other known failure modes?
+
+feature gate disabled: the API will always return a well-known error. In normal operation, the API is expected to never return error and always return
+a valid response, because it utilizes internal kubelet data which is always available.
+Bugs may lead to the API to return unexpected errors, or to return inconsistent data.
+Consumers of the API should treat unexpected errors as bugs of this API.
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
 
-[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
-[existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
+Not available.
```

Comment:
If users of this feature do encounter unexpected errors and/or inconsistent data, what actions should they/can they take to troubleshoot the unexpected errors and remediate the issues?  Is it possible that end users can see any errors here as a result of the rate limiting introduced as part of the pod resources endpoint GA graduation?

Reply:
Good point. Clarified in the text.
Since the API exposes internal kubelet data (ofc in read only) pertaining to resource allocation, it can be a early warning (or a proxy messenger) of kubelet allocation issues. In this case the only possible investigation is to check kubelet logs. Amended the KEP text to reflect this fact